test/cpp/api/nn_utils.cpp

```diff
@@ -101,3 +101,41 @@ TEST_F(NNUtilsTest, ClipGradNorm) {
     ASSERT_TRUE(torch::allclose(p1.grad(), p2.grad()));
   }
 }
+TEST_F(NNUtilsTest, ClipGradValue) {
+  auto linear_layer = Linear(10, 10);
+  float clip_value = 2.5;
+
+  torch::Tensor grad_w = torch::arange(-50., 50).view({10, 10}).div_(5);
+  torch::Tensor grad_b = torch::ones(10).mul_(2);
```

Comment:
I think we might need to write:
```suggestion
  torch::Tensor grad_b = torch::ones({10}).mul_(2);
```